README.md

```diff
@@ -1 +1,59 @@
-# School-Website
\ No newline at end of file
+## SCHOOL-WEBSITE
+
+![Elite school logo](img/logo.png)
+
+`Elite school` is the best school to nurture you child and enlight him/her academically.
+## Desktop Version
+![Desktop Version](img/desktop1.png)
+![Desktop Version](img/desktop2.png)
+![Desktop Version](img/desktop3.png)
+![Desktop Version](img/desktop4.png)
+## Mobile version
+![Desktop Version](img/mob1.png)
+![Desktop Version](img/mob2.png)
+![Desktop Version](img/mob3.png)
+![Desktop Version](img/mob4.png)
+## Usage
+
+---
+
+```
+Usage: index.html [OPTIONS]
+
+  Securely configure your computer.
+  Developed by Josphat Kiploman -> (Github: Josphat205),
+ - (Twitter: Josphat),
+ - (Linkedin: Josphat kiploman)
+
+
+Options:
+  -lockdown  Set secure configuration without user interaction.
+  -v         Display version and author information and exit.
+  -help, -h  Show this message and exit.
+```
+
+## **Installation Options**
+
+1. Install with [`git`](git@github.com:Josphat205/School-Website.git)
+
+   - `$ git clone git@github.com:Josphat205/School-Website.git`
+
+2. Download the `Elite School website` binary from Releases tab.
+
+## **How to Contribute**
+
+1. Clone repo and create a new branch: `$ git clone git@github.com:Josphat205/School-Website.git`.
+2. Make changes and test
+3. Submit Pull Request with comprehensive description of changes
+
+## Link to the Video
+[![Video](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)
+
+## **Acknowledgements**
```

Comment:
- [ ] In your  README  kindly  give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).